stl/inc/bit

```diff
@@ -84,7 +84,14 @@ _NODISCARD constexpr int countl_zero(_Ty _Val) noexcept;
 
 _EXPORT_STD template <_Standard_unsigned_integral _Ty>
 _NODISCARD constexpr bool has_single_bit(const _Ty _Val) noexcept {
+#if defined(_POPCNT_INTRINSICS_ALWAYS_AVAILABLE) && _POPCNT_INTRINSICS_ALWAYS_AVAILABLE
+    if (_STD is_constant_evaluated()) {
+        return (_Val ^ (_Val - 1)) > _Val - 1;
+    }
+    return _Unchecked_popcount(_Val) == 1;
+#else

```

Comment:
We have standard `#else` comments like 
```c++
// ^^^ defined(_POPCNT_INTRINSICS_ALWAYS_AVAILABLE) && _POPCNT_INTRINSICS_ALWAYS_AVAILABLE / !defined(_POPCNT_INTRINSICS_ALWAYS_AVAILABLE) || !_POPCNT_INTRINSICS_ALWAYS_AVAILABLE vvv
```

But that looks too verbose for this case

Maybe 

```
// ^^^ popcnt intrinsic always available / popcnt intrinsic not available or not always available vvv
```
The `#endif` comment should consistently say
```c++
// ^^^ popcnt intrinsic not available or not always available ^^^
```

Reply:
updated